examples/nextjs/app/page.tsx

```diff
@@ -1,5 +1,5 @@
 "use client";
-import initModule, { Workbook } from "wasm-xlsxwriter";
+import initModule, { Workbook } from "../../../pkg.web/";
```

Comment:
Would you mind avoiding the use of absolute paths and using package names instead? We'd like to ensure that the package can be imported correctly. Could you please use something like `wasm-xlsxwriter/client`?